backend/app/main/main.py

```diff
@@ -272,3 +272,19 @@ def create_user(user: schemas.UserCreate, session: Session = Depends(get_db)):
     :return: JSON response with created entry
     """
     return crud.create_user(session=session, user=user)
+
+
+@router.get(
+    "/get_session_food_items/{food_collect_id}", response_model=List[schemas.Food]
+)
+def get_session_food_items(food_collect_id, session: Session = Depends(get_db)):
+    """
+    Get session food items from the database - MealCare
+
+    :return: JSON response with get all entries
+    """
+
+    db_session_food_items = crud.get_session_food_items(session, food_collect_id)
```

Comment:
nice job, maybe just change the crud method call to match the rest of the endpoints i.e)crud.get_session_food_items(session: Sessio, food_collect_id=food_collect_id)